bigtable/connectionpool/configure_connection_pool.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START bigtable_configure_connection_pool]
+
+// Connectionpool is a sample program demonstrating how to configure the number
```

Comment:
```suggestion
// Package connectionpool demonstrates how to configure the number
```

https://github.com/golang/go/wiki/CodeReviewComments#package-comments